test/utils/pki_helpers.go

```diff
@@ -23,12 +23,11 @@ import (
 	"crypto/x509"
 	"crypto/x509/pkix"
 	"encoding/pem"
+	"errors"
 	"math"
 	"math/big"
 	"time"
 
-	"github.com/pkg/errors"
```

Comment:
you have to replace Lines 61 and 64 before removing the imported library

Reply:
thx! replaced now, feel sorry about the mistake 😅

Reply:
no worries, that happens to all of us